DBlock Game/DBlock Game/Program.cs

```diff
@@ -1347,6 +1355,133 @@ public static void Fight()
+
+                switch (num2)
+                {
+                    case 1:
+                        comp = "PAPER";
+                        break;
+                    case 2:
+                        comp = "SCISSORS";
+                        break;
+                    case 3:
+                        comp = "ROCK";
+                        break;
+                }
+
+                // 
+                switch (player)
+                {
+                    case "PAPER":
+                        if (comp == "PAPER")
+                        {
+                            pscore++;
+                            cscore++;
+                            TextBox("Its a draw you both score a point.", Player, false);
+                        }
+                        else if (comp == "SCISSORS")
+                        {
+                            cscore++;
+                            TextBox("Vaugh won he gains a point.", Player, false);
+                        }
+                        else
+                        {
+                            pscore++;
+                            TextBox("You won you gain a point", Player, false);
+                        }
+                        break;
+
+                    case "SCISSORS":
+                        if (comp == "SCISSORS")
+                        {
+                            pscore++;
+                            cscore++;
+                            TextBox("Its a draw you both score a point.", Player, false);
+                        }
+                        else if (comp == "ROCK")
+                        {
+                            cscore++;
+                            TextBox("Vaugh won he gains a point.", Player, false);
+                        }
+                        else
+                        {
+                            pscore++;
+                            TextBox("You won you gain a point", Player, false);
+                        }
+                        break;
+
+                    case "ROCK":
+                        if (comp == "ROCK")
+                        {
+                            pscore++;
+                            cscore++;
+                            TextBox("Its a draw you both score a point.", Player, false);
+                        }
+                        else if (comp == "PAPER")
+                        {
+                            cscore++;
+                            TextBox("Vaugh won he gains a point.", Player, false);
+                        }
+                        else
+                        {
+                            pscore++;
+                            TextBox("You won you gain a point", Player, false);
+                        }
+                        break;
+
+                }
+
+            }
+
+            //this takes them back to the load save if they fail
+            if( cscore > pscore)
+            {
+                Loadgame();
+            }
+
```

Comment:
a few lines of white space but i would merge seems like everything works, and nothing game breaking